helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -743,6 +743,8 @@ private void createZKPaths(String clusterName) {
     _zkClient.createPersistent(path);
     path = PropertyPathBuilder.resourceConfig(clusterName);
     _zkClient.createPersistent(path);
+    path = PropertyPathBuilder.customizedStateConfig(clusterName);
```

Comment:
This is OK for this PR, but ideally, we should separate this type of API change into other PRs.

Reply:
Is this change covered by any unit tests?

Reply:
added in TestZkHelixAdmin